.gitignore

```diff
@@ -10,3 +10,5 @@ Homestead.json
 Homestead.yaml
 npm-debug.log
 yarn-error.log
+composer.lock
+
```

Comment:
Do not edit .gitignore

Reply:
But the composer.lock is a file that is auto-generated. You didn't put it on .gitignore

Reply:
Having composer.lock on repository ensures we all have the same versions unless someone updates them

Reply:
I change the .gitignore file to exclude that file